src/Configuration.php

```diff
@@ -38,67 +43,67 @@ public function apply(Injector $injector, array $env)
         $services = "Wheniwork\Feedback\Service";
```

Comment:
Instead of using the string aliases, you could also do:

```php
use Wheniwork\Feedback\Service;

// ...

$injector->define(Service\Authorizer::class, ...);
// etc
```